project_1.html

```diff
@@ -0,0 +1,421 @@
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-concepts-ohms-law">Ohm's Law</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-concepts-resistor">Resistor</a>
+							</li>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-concepts-led">LED (Light Emitting Diode)</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-concepts-slide-switch">Slide Switch</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-concepts-soldering">Soldering</a>
+							</li>
+						</ul>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-requirements">Requirements</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-parts">Parts</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-schematics">Schematics</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-instructions">Instructions</a>
+						<ul class="nav">
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-checkpoint1">Checkpoint 1</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-checkpoint2">Checkpoint 2</a>
+							</li>
+							<li class="nav-item">
+								<a class="nav-link" href="#pj-heading-checkpoint3">Checkpoint 3</a>
+							</li>
+						</ul>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#pj-heading-deliverables">Deliverables</a>
+					</li>
+				</ul>
+			</nav>
+		</aside>
+		<!-- End Sidebar -->
+
+		<main class="pj-main p-3" data-bs-spy="scroll" data-bs-target="#project-toc" data-bs-root-margin="0px 0px -40%"
+			data-bs-smooth-scroll="true" tabindex="0">
+			<div class="d-flex flex-row justify-content-between mb-5">
+				<div class="">
+					<h4>Project 1</h4>
+					<h1 class="fw-bold">LED There Be Light</h1>
+				</div>
+				<div class="alert alert-light" role="alert">
+					<i class="bi bi-calendar-event me-1"></i>
+					Updated 2023.09.20 <br>
+					By Gavin Nguyen and Dylan Vu
+				</div>
+			</div>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-introduction">Overview</h2>
+			<p class="mb-5">
+				Every electrical engineer has to start somewhere; for you, it’s no different. We’ll stick with the classic beginner’s project:
+				You will build a basic LED circuit on a breadboard. Then, you will solder it onto a perfboard.
+				<br><br>
+				Because this is the very first project, let’s walk through the layout of this webpage: The first section, “Concepts”, discusses
+				the key details from the accompanying lecture that are most relevant to the project. The following section, “Requirements”, lists
+				all the <i>requirements</i> your deliverable must meet for full marks.  The “Parts” and “Schematics” sections show the components needed
```

Comment:
The \<i\> tag lowers the css font weight for some unexpected reason. I think it should be the same as the font weight as the rest of the text. Please investigate a fix for this. I suspect we may just have to override it by writing our own CSS rules for italics.

Reply:
fixed, we think it's because of the selected font weights imported in the head area. Need to propagate this fix among the current PRs, I fixed it for all the other files by importing the entirety of roboto